packages/react-conformance/package.json

```diff
@@ -1,7 +1,6 @@
 {
   "name": "@fluentui/react-conformance",
-  "version": "1.0.0",
-  "private": true,
+  "version": "0.2.0",
```

Comment:
One more thing here, sorry: can you add the following config at the bottom of this file?
```
  "beachball": {
    "tag": "latest",
    "disallowedChangeTypes": [
      "major",
      "prerelease"
    ]
  }
```